tolerance-policies/apps/monitor/src/main/java/com/redhat/training/sysinfo/InfoService.java

```diff
@@ -0,0 +1,32 @@
+package com.redhat.training.sysinfo;
+
+import io.quarkus.logging.Log;
+import javax.enterprise.context.ApplicationScoped;
+import org.eclipse.microprofile.faulttolerance.Retry;
+
+/**
+ * Simulate a microservice that reads system info from a cloud instance
+ */
+@ApplicationScoped
+public class InfoService {
+
+    private int callCount = 0;
+
+    public Info getInfo() {
+        callCount++;
+
+        crashPossibly();
+
+        return new Info();
+    }
+
+    private void crashPossibly() {
+        if ( callCount % 5 > 0 ) {
+            Log.error( "Request #" + callCount + " has failed" );
+            throw new RuntimeException( "InfoService failed due to unexpected error" );
+        } else {
+            Log.info( "Request #" + callCount + " has succeeded" );
+        }
```

Comment:
Just like above, this would be easy to flip. Essentially, "every 5th request succeeds, otherwise fail" instead of "every request that is not fifth fails, otherwise succeed". 

I think flipping the condition in this particular case helps a lot with readability.

WDYT?

```suggestion
        if ( callCount % 5 == 0 ) {
            Log.info( "Request #" + callCount + " has succeeded" );
        } else {
           Log.error( "Request #" + callCount + " has failed" );
           throw new RuntimeException( "InfoService failed due to unexpected error" );
        }
```